sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -287,6 +290,11 @@ abstract class StreamExecution(
       sparkSession.sparkContext.setJobGroup(runId.toString, getBatchDescriptionString,
         interruptOnCancel = true)
       sparkSession.sparkContext.setLocalProperty(StreamExecution.QUERY_ID_KEY, id.toString)
+      loggingThreadContext = CloseableThreadContext.putAll(
+        Map(
+          StreamExecution.QUERY_ID_KEY -> id.toString,
```

Comment:
Shall we use/create keys from LogKey.scala as well? We can have `streaming_query_id`(or just query_id) and `streaming_run_id`.
It would be great if we can keep the variable names consistent.

Reply:
oh yes I was using that, forgot to push update to github!